src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_source_configuration/prometheus/configure_prometheus_data_source.tsx

```diff
@@ -88,9 +87,9 @@ export const ConfigurePrometheusDatasourcePanel = (props: ConfigurePrometheusDat
   return (
     <div>
       <EuiPanel>
-        <EuiTitle>
-          <h4>{`Configure Prometheus data source`}</h4>
-        </EuiTitle>
+        <EuiText size="s">
+          <h1>{`Configure Prometheus data source`}</h1>
```

Comment:
is this a page title?

Reply:
It's the title of this page for creating a Prometheus data source: 
<img width="1030" alt="Screenshot 2024-08-05 at 2 53 47 PM" src="https://github.com/user-attachments/assets/af636da1-591f-4dd2-b557-285b009fbcaa">